src/views/HowToPlay/Layout/InstructionsLayout.tsx

```diff
@@ -2,11 +2,13 @@ import { KeepReading, BackButton } from '@ui/components';
 import { Outlet } from 'react-router';
 import StyledInstructionsLayout from './StyledInstructionsLayout';
 import { StyledInstructionsBody } from './StyledInstructionsBody';
+import { TitleBanner } from '@ui/components';
 
 export const InstructionsLayout = () => {
   return (
     <>
       <StyledInstructionsLayout>
+        <TitleBanner title="How To Play"></TitleBanner>
```

Comment:
Suggestion for future refactor. It's kind of odd to see the innerHTML of the component as a prop instead of content. You might consider just getting rid of the prop in the future.

```suggestion
        <TitleBanner>How To Play</TitleBanner>
```